libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/NotificationRenderer.kt

```diff
@@ -41,24 +44,37 @@ class NotificationRenderer(
     ) {
         val color = enterpriseService.brandColorsFlow(currentUser.userId).first()?.toArgb()
             ?: NotificationConfig.NOTIFICATION_ACCENT_COLOR
+        val numberOfAccounts = sessionStore.getAllSessions().size
```

Comment:
We are getting all the session data each time we render a notification just to get the number of sessions? Not ideal